typescript/infra/src/agents/key-utils.ts

```diff
@@ -19,20 +19,41 @@ interface KeyAsAddress {
   address: string;
 }
 
+export function getRelayerCloudAgentKeys(
+  agentConfig: AgentContextConfig,
+): Array<CloudAgentKey> {
+  const keys = [];
+
+  const isAws = !!agentConfig.aws;
+  if (isAws) {
+    keys.push(new AgentAwsKey(agentConfig, Role.Relayer));
+    let nonEthereumChains = agentConfig.contextChainNames.find(
+      (chainName) =>
+        chainMetadata[chainName].protocol !== ProtocolType.Ethereum,
+    );
+    // If there are any non-ethereum chains, we also want hex keys.
+    if (nonEthereumChains) {
+      keys.push(
+        new AgentGCPKey(agentConfig.runEnv, agentConfig.context, Role.Relayer),
+      );
+    }
+  } else {
+    keys.push(
+      new AgentGCPKey(agentConfig.runEnv, agentConfig.context, Role.Relayer),
+    );
+  }
+
+  return keys;
+}
+
 export function getCloudAgentKey(
   agentConfig: AgentContextConfig,
   role: Role,
```

Comment:
I would like to see it clear that the relayer role should go elsewhere.
```suggestion
  role: Role.Validator | Role.Deployer | Role.Kathy | Role.Bank,
```

Reply:
This breaks some other usage of `getCloudAgentKey` where you want a specific key. Added a comment to clarify intentions